app/services/db/proofing_cost/add_user_proofing_cost.rb

```diff
@@ -15,7 +16,10 @@ class AddUserProofingCost
       def self.call(user_id, token)
         return unless user_id
         proofing_cost = ::ProofingCost.find_or_create_by(user_id: user_id)
-        return unless TOKEN_WHITELIST.index(token.to_sym)
+        unless TOKEN_WHITELIST.index(token.to_sym)
```

Comment:
```suggestion
        unless TOKEN_WHITELIST.include?(token.to_sym)
```

What about `.include?` to make it clearer it's a membership check?